app/main.py

```diff
@@ -60,6 +60,10 @@ def disconnect_db():
 
 if __name__ == "__main__":
     import uvicorn  # type: ignore
+    import json
+
+    print('Environment:')
+    print(json.dumps(settings.dict(), indent=4, sort_keys=True))
```

Comment:
Debugging leftover?

Reply:
Ehh i think logging that is kinda helpful :v (intentional)

Reply:
This will leak secrets (`CONSUMER_KEY`, ...)

Reply:
I dont think logging inside a container counts as leaking right?

Reply:
Why not?

Reply:
cause if anyone have access to the container they would already have the .env file? how could some logs compare to that?

Reply:
But there is no `.env` file in container?

Reply:
yea no, so how could it even leak without access to the host?